25.0.5 breaks CLI mode

### Description

It seems that 25.0.5 introduces a regression in the CLI mode of the application e.g. you can't launch dbeaver-ce --help anymore without the GUI being brought up as well.


### DBeaver Version

Community Edition 25.0.5

### Operating System

Rocky Linux 8

### Database and driver

irrelevant

### Steps to reproduce

With version 25.0.4 everything regarding the CLI mode is working fine.

```
$ docker run -it rockylinux:8 bash -c 'curl -sLo db.rpm https://dbeaver.io/files/25.0.4/dbeaver-ce-25.0.4-stable.x86_64.rpm && dnf install -y db.rpm && dbeaver-ce --version || cat /usr/share/dbeaver-ce/configuration/*.log'
Rocky Linux 8 - AppStream                                                                                                                                                   15 MB/s |  18 MB     00:01    
Rocky Linux 8 - BaseOS                                                                                                                                                      10 MB/s |  23 MB     00:02    
Rocky Linux 8 - Extras                                                                                                                                                      42 kB/s |  15 kB     00:00    
Dependencies resolved.
===========================================================================================================================================================================================================
 Package                                         Architecture                                Version                                               Repository                                         Size
===========================================================================================================================================================================================================
Installing:
 dbeaver-ce                                      x86_64                                      25.0.4-stable                                         @commandline                                      126 M

Transaction Summary
===========================================================================================================================================================================================================
Install  1 Package

Total size: 126 M
Installed size: 164 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                   1/1 
  Installing       : dbeaver-ce-25.0.4-stable.x86_64                                                                                                                                                   1/1 
  Running scriptlet: dbeaver-ce-25.0.4-stable.x86_64                                                                                                                                                   1/1 
  Verifying        : dbeaver-ce-25.0.4-stable.x86_64                                                                                                                                                   1/1 

Installed:
  dbeaver-ce-25.0.4-stable.x86_64                                                                                                                                                                          

Complete!
DBeaver 25.0.4.202505041731
```
----

Now with 25.0.5 it appears that DBeaver attempts to launch the graphical user interface which is clearly not required here. DBeaver should not attempt to load GUI libs like gtk3 but simply print the version and exit immediately.

```
$ docker run -it rockylinux:8 bash -c 'curl -sLo db.rpm https://dbeaver.io/files/25.0.5/dbeaver-ce-25.0.5-stable.x86_64.rpm && dnf install -y db.rpm && dbeaver-ce --version || cat /usr/share/dbeaver-ce/configuration/*.log'
Rocky Linux 8 - AppStream                                                                                                                                                   13 MB/s |  18 MB     00:01    
Rocky Linux 8 - BaseOS                                                                                                                                                      14 MB/s |  23 MB     00:01    
Rocky Linux 8 - Extras                                                                                                                                                      41 kB/s |  15 kB     00:00    
Dependencies resolved.
===========================================================================================================================================================================================================
 Package                                         Architecture                                Version                                               Repository                                         Size
===========================================================================================================================================================================================================
Installing:
 dbeaver-ce                                      x86_64                                      25.0.5-stable                                         @commandline                                      125 M

Transaction Summary
===========================================================================================================================================================================================================
Install  1 Package

Total size: 125 M
Installed size: 163 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                   1/1 
  Installing       : dbeaver-ce-25.0.5-stable.x86_64                                                                                                                                                   1/1 
  Running scriptlet: dbeaver-ce-25.0.5-stable.x86_64                                                                                                                                                   1/1 
  Verifying        : dbeaver-ce-25.0.5-stable.x86_64                                                                                                                                                   1/1 

Installed:
  dbeaver-ce-25.0.5-stable.x86_64                                                                                                                                                                          

Complete!
SWT OS.java Error: Failed to load swt-pi3, loading swt-pi4 as fallback.
Dbeaver:
An error has occurred. See the log file
/usr/share/dbeaver-ce/configuration/1747992820134.log.
!SESSION 2025-05-23 09:33:39.861 -----------------------------------------------
eclipse.buildId=unknown
java.version=21.0.5
java.vendor=Eclipse Adoptium
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Framework arguments:  --version
Command-line arguments:  -os linux -ws gtk -arch x86_64 --version

!ENTRY org.eclipse.osgi 4 0 2025-05-23 09:33:42.481
!MESSAGE An error occurred while automatically activating bundle org.eclipse.ui.ide (163).
!STACK 0
org.osgi.framework.BundleException: Exception in org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start() of bundle org.eclipse.ui.ide.
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:847)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:768)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1066)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:394)
	at org.eclipse.osgi.container.Module.doStart(Module.java:643)
	at org.eclipse.osgi.container.Module.start(Module.java:500)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:528)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:122)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:622)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:353)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:521)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:438)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:195)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.getRecentWorkspaces(DBeaverApplication.java:388)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.setIDEWorkspace(DBeaverApplication.java:364)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:203)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:149)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:115)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:467)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:298)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:797)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:635)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1578)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.main(DBeaverLauncher.java:1551)
Caused by: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory
	no swt-pi3-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi3 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3.so
	/root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory

	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
	at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:99)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:209)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:155)
	at org.eclipse.swt.widgets.Display.<clinit>(Display.java:170)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.createProblemsViews(IDEWorkbenchPlugin.java:390)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start(IDEWorkbenchPlugin.java:347)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:826)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:818)
	... 29 more
	Suppressed: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory
	no swt-pi4-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi4 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4.so
	/root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory

		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
		at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:103)
		... 38 more
Root exception:
java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory
	no swt-pi3-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi3 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3.so
	/root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory

	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
	at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:99)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:209)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:155)
	at org.eclipse.swt.widgets.Display.<clinit>(Display.java:170)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.createProblemsViews(IDEWorkbenchPlugin.java:390)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start(IDEWorkbenchPlugin.java:347)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:826)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:818)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:768)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1066)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:394)
	at org.eclipse.osgi.container.Module.doStart(Module.java:643)
	at org.eclipse.osgi.container.Module.start(Module.java:500)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:528)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:122)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:622)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:353)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:521)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:438)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:195)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.getRecentWorkspaces(DBeaverApplication.java:388)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.setIDEWorkspace(DBeaverApplication.java:364)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:203)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:149)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:115)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:467)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:298)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:797)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:635)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1578)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.main(DBeaverLauncher.java:1551)
	Suppressed: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory
	no swt-pi4-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi4 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4.so
	/root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory

		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
		at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:103)
		... 38 more

!ENTRY org.eclipse.osgi 4 0 2025-05-23 09:33:42.549
!MESSAGE Application error
!STACK 1
java.lang.NoClassDefFoundError: org/eclipse/ui/internal/ide/ChooseWorkspaceData
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.getRecentWorkspaces(DBeaverApplication.java:388)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.setIDEWorkspace(DBeaverApplication.java:364)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:203)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:149)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:115)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:467)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:298)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:797)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:635)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1578)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.main(DBeaverLauncher.java:1551)
Caused by: java.lang.ClassNotFoundException: An error occurred while automatically activating bundle org.eclipse.ui.ide (163).
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:134)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:622)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:353)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:417)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:521)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:438)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:195)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 14 more
Caused by: org.osgi.framework.BundleException: Exception in org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start() of bundle org.eclipse.ui.ide.
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:847)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:768)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1066)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:394)
	at org.eclipse.osgi.container.Module.doStart(Module.java:643)
	at org.eclipse.osgi.container.Module.start(Module.java:500)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:528)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:122)
	... 22 more
Caused by: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory
	no swt-pi3-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi3 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi3.so
	/root/.swt/lib/linux/x86_64/libswt-pi3-gtk-4968r13.so: libgtk-3.so.0: cannot open shared object file: No such file or directory

	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
	at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
	at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:99)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:209)
	at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:155)
	at org.eclipse.swt.widgets.Display.<clinit>(Display.java:170)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.createProblemsViews(IDEWorkbenchPlugin.java:390)
	at org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start(IDEWorkbenchPlugin.java:347)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:826)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:818)
	... 29 more
	Suppressed: java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons: 
	/usr/share/dbeaver-ce/configuration/org.eclipse.osgi/153/0/.cp/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory
	no swt-pi4-gtk in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	no swt-pi4 in java.library.path: /usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4-gtk.so
	Can't load library: /root/.swt/lib/linux/x86_64/libswt-pi4.so
	/root/.swt/lib/linux/x86_64/libswt-pi4-gtk-4968r13.so: libgtk-4.so.1: cannot open shared object file: No such file or directory

		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:345)
		at org.eclipse.swt.internal.Library.loadLibrary(Library.java:254)
		at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:103)
		... 38 more
```



### Additional context

We have an automation process in place which verifies each new version of the DBeaver RPMs prior to releasing it to our user workstations. This automation failed with the new version 25.0.5 right away because it was unable to determine the version number by using `dbeaver-ce --version`

We clearly consider this a problem of the new version 25.0.5.

There was a similar bug in the past with #36061.

We recommend to have the CLI mode added to the DBeaver QA stage of release preparation.

--- a/plugins/org.jkiss.dbeaver.model.cli/src/org/jkiss/dbeaver/model/cli/ApplicationCommandLine.java
+++ b/plugins/org.jkiss.dbeaver.model.cli/src/org/jkiss/dbeaver/model/cli/ApplicationCommandLine.java
@@ -30,7 +30,9 @@ import org.jkiss.utils.CommonUtils;
 
 import java.io.PrintWriter;
 import java.io.StringWriter;
-import java.util.*;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
 import java.util.stream.Collectors;
 
 public abstract class ApplicationCommandLine<T extends ApplicationInstanceController> {
@@ -66,11 +68,6 @@ public abstract class ApplicationCommandLine<T extends ApplicationInstanceContro
             return new CmdProcessResult(CmdProcessResult.PostAction.START_INSTANCE);
         }
 
-        if (controller == null) {
-            log.debug("Can't process commands because no running instance is present");
-            return new CmdProcessResult(CmdProcessResult.PostAction.START_INSTANCE);
-        }
-
         if (commandLine.hasOption(PARAM_HELP)) {
             HelpFormatter helpFormatter = new HelpFormatter();
             helpFormatter.setWidth(120);
@@ -90,18 +87,22 @@ public abstract class ApplicationCommandLine<T extends ApplicationInstanceContro
             }
         }
 
-        if (!uiActivated) {
-            if (commandLine.hasOption(PARAM_THREAD_DUMP)) {
-                String threadDump = controller.getThreadDump();
-                System.out.println(threadDump);
-                return new CmdProcessResult(CmdProcessResult.PostAction.SHUTDOWN, threadDump);
-            }
-
-        }
         if (commandLine.hasOption(PARAM_VERSION)) {
             String version = GeneralUtils.getLongProductTitle();
             System.out.println(version);
             return new CmdProcessResult(CmdProcessResult.PostAction.SHUTDOWN, version);
+        }
+
+        if (!uiActivated) {
+            if (commandLine.hasOption(PARAM_THREAD_DUMP)) {
+                if (controller == null) {
+                    log.debug("Can't process commands because no running instance is present");
+                    return new CmdProcessResult(CmdProcessResult.PostAction.START_INSTANCE);
+                }
+                String threadDump = controller.getThreadDump();
+                System.out.println(threadDump);
+                return new CmdProcessResult(CmdProcessResult.PostAction.SHUTDOWN, threadDump);
+            }
         }
 
         return new CmdProcessResult(CmdProcessResult.PostAction.UNKNOWN_COMMAND);

